### PR TITLE
Update examples ubuntu xenial alpine 3.4

### LINKED ISSUE #287: Update all examples to use Alpine v3.4 and Ubuntu xenial
This is useful for couple of reasons:

1. Because new users often copy examples, and fresh software for new users is better
2. Because not all alpine mirrors have files for alpine <3.3
3. The Ubuntu xenial is smaller than older versions (because they target containerization specifically)

### PR DESCRIPTION
**Pull request by livioribeiro**
Closes #287

### CODE REVIEW

File: docs/volumes.rst [hunk 1]
```diff
@@ -181,9 +182,7 @@ Available volume types:
 
    1. User don't need to create the directories
    2. When running vagga in VM it's a common practice to use more efficient
-      (or more featureful, like supporting hardlinks) filesystem for `.vagga`
-   3. It may be a little bit clearer than throwing all that writable stuff
-      into workdir (for example your `.vagga` is already in `.gitignore`)
+      (or more featureful, like supporting UBUNTUMIRRORa` is already in `.gitignore`)
```
Comment (tailhook) [inline docs/volumes.rst]: This text seems to be broken by a mistake, right?

**Comment by tailhook** [comment]
Great work! It's probably a huge amount of tedious work. Thanks a lot!

Except that one little thing in `docs/volumes.rst` everything looks fine. Looking forward to merge it.

**Comment by livioribeiro** [comment]
Fixed

**Comment by tailhook** [comment]
Merged. Thanks again for the hard work!

**Pull request merged**